lib/ma_crossover/meta/gen_order_label.js

```diff
@@ -13,10 +13,11 @@
 const genOrderLabel = (state = {}) => {
   const { args = {} } = state
   const { orderType, orderPrice, amount, long, short } = args
+  const priceShouldBeDisplayed = orderType === 'LIMIT'
 
   return [
     'MA Crossover',
-    ` | ${amount} @ ${orderPrice || orderType} `,
+    ` | ${amount} @ ${priceShouldBeDisplayed ? orderPrice : orderType} `,
```

Comment:
May be we should add the following in `process_params.js` as well so that we dont save it in db. What do you think @vitaliystoliarovcc ?
```
if (params.orderType !== 'LIMIT') {
    params.orderPrice = 0
}
```

Reply:
I don’t think this is necessary, as it is assumed that orderPrice in the Birfinex API will be ignored if orderType !== 'LIMIT'. @robertkowalski, isn't?

Reply:
> as well so that we don't save it in db

can you point where this data is stored in the database?

Reply:
> can you point where this data is stored in the database?

```
~/.honeyframework/algos.json
```

is a json database of algo orders. it contains the data serialized in `meta/serialize.js`

Reply:
> I don’t think this is necessary, as it is assumed that orderPrice in the Birfinex API will be ignored if orderType !== 'LIMIT'. @robertkowalski, isn't?

hmmm... in the first place i was wondering why the ui allows to submit market orders with a price...? then as second thought i think the cleaner approach is to fix it in process params, if we receive invalid data from the ui. for the feature the current approach also makes things working but why should we keep invalid values?